content/client-server-api/modules/mentions.md

```diff
@@ -1,75 +1,59 @@
 
 ### User and room mentions
 
-This module allows users to mention other users and rooms within a room message.
+{{< changed-in v="1.7" >}}
```

Comment:
sorry, I think this should have been %s to use the right spacing (for reasons I have never quite taken the time to grok)

```suggestion
{{% changed-in v="1.7" %}}
```

Reply:
similarly for the push rules